src/main/java/com/minecolonies/core/event/EventHandler.java

```diff
@@ -136,6 +147,67 @@ public static void onEntityAdded(@NotNull final EntityJoinLevelEvent event)
         }
     }
 
+    @SubscribeEvent
+    public static void onLootTableLoad(@NotNull final LootTableLoadEvent event)
+    {
+        if (event.getName().toString().equals("minecraft:blocks/fern"))
```

Comment:
Maybe we should also add some seed as a dungeon loot to simpledungeon chest or so, like pumpkin seeds are in vanilla